docs/custom-music/index.md

```diff
@@ -6,12 +6,12 @@ sidebar_position: 3
 
 ## Prerequisites
 :::note
-You do not require Unreal Engine to be installed to replace the game's audio
+You do not require Unreal Engine to be installed to replace the game's audio (But if you seriously want to, double check the engine version and installation options not path option, that will save your time)
```

Comment:
I see no need for this extra comment, it will only make people think it is possible to use unreal engine for this. It might be, but it's an almost completely different process that has not been documented here yet.